packages/elements-core/src/components/Docs/HttpService/SecuritySchemes.tsx

```diff
@@ -27,44 +27,47 @@ export const SecuritySchemes: React.FC<SecuritySchemesProps> = ({
         </Box>
       </Panel.Titlebar>
       <Panel.Content p={0}>
-        {sortBy(schemes, 'type').map((scheme, i) => (
-          <SecurityScheme
-            key={i}
-            scheme={scheme}
-            defaultIsOpen={defaultScheme ? scheme.key === defaultScheme : i === 0}
-            isCollapsible={schemes.length > 1}
-            showSchemeKey={shouldIncludeKey(schemes, scheme.type)}
-          />
-        ))}
+        {includeOptional && <OptionalMessage />}
+        {secSchemes
```

Comment:
what about `NodeAnnotation` for the top-level security panel? I suppose right now you won't see which security changed after you expand the panel.
We'd probably need to put it next to the Panel

Reply:
Hmmm... is this the http-security-group node then?  I still don't know how I'm supposed to recover that node id and make it match to Billie's work in sl-doc - it looked like she was using the parent node id 🤔 would that be the ID for the overall service - will I have access to that in the specs we're representing?

Reply:
> is this the http-security-group node then? 

right, that'd be it.
I think you'll need to reconstruct it, kind of like we do it here https://github.com/stoplightio/json-schema-viewer/blob/431fc71c26c8621141d6ed6333350cc70eafcfac/src/hash.ts#L18
the `parentId` in this case would be the id of the entire document, so we should have it in theory.
In the case of operation, this would be the id of the operation which should also be there.

Reply:
Ok - take a look.  There is something Billie was doing in stoplight-doc - I'm not sure why?  It's the whole if withURI use props.uri except... that prop will never be there?  For a group I mean, since it's an array.  🤔 

Anyhow, just did what I expect her stoplight-doc to do, which is fall back on the parentId and position in the array each time.  Take a look, let me know what you think :) - thanks!

Reply:
na, the URI won't be there, can ignore that bit